src/components/organism/ProjectForm/index.jsx

```diff
@@ -77,21 +90,37 @@ const ProjectForm = () => {
   });
 
   const formInitialValues = {
-    projectTitle: '',
-    clientId: '',
-    clientName: '',
-    clientDescription: '',
-    projectDescription: '',
-    projectOutcomes: '',
-    coverImageUrl: '',
-    projectAndis: [],
-    projectTech: [],
+    projectTitle: 'r',
+    clientId: 'r',
+    clientName: 'r',
+    clientDescription: 'r',
+    projectDescription: 'r',
+    projectOutcomes: 'r',
+    coverImageUrl: 'r',
+    projectAndis: [{ id: 1, name: 'James'}],
+    projectTech: [{id: 1, name: 'React'}],
     currentAndiName: '',
     currentTechName: ''
```

Comment:
why are these values here, should they not be blank?

Reply:
These are default values for testing! Good spot though. The post request isn't actually working yet - working on this in a different PR!